core/src/plugin/index.ts

```diff
@@ -23,4 +24,5 @@ export interface Summary {
 
 export const PLUGINS: Record<string, Plugin> = {
   openai: openai as unknown as Plugin,
+  bedrock: bedrock as unknown as Plugin
```

Comment:
Could you start a README section for the engines we have?

Reply:
What do you want in it?

Reply:
You got it right